app/move/fields/create.js

```diff
@@ -307,7 +307,7 @@ module.exports = {
     fieldset: {
       legend: {
         text: 'fields::date_type.label',
-        classes: 'govuk-fieldset__legend--m',
+        classes: 'govuk-visually-hidden govuk-fieldset__legend--m',
```

Comment:
Isn't the govuk design system way actually to set a`isPageHeading` property to `true` on the `legend`?

Reply:
Yep, good spot. That is the way they suggest doing it now. But that might need some re-working of the current templates.

Maybe one to add to the issues?